nixos/lib/utils.nix

```
@@ -197,5 +197,6 @@ rec {
   systemdUtils = {
     lib = import ./systemd-lib.nix { inherit lib config pkgs; };
     unitOptions = import ./systemd-unit-options.nix { inherit lib systemdUtils; };
+    types = import ./systemd-types.nix { inherit lib systemdUtils; };
```

```suggestion
    lib = import ./systemd/lib.nix { inherit lib config pkgs; };
    unitOptions = import ./systemd/unit-options.nix { inherit lib systemdUtils; };
    types = import ./systemd/types.nix { inherit lib systemdUtils; };
```
Maybe? I am not sure. Just an idea.